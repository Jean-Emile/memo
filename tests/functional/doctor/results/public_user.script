{"drives":[],"leftovers":[],"networks":[],"sane":false,"storage resources":[],"volumes":[]}
