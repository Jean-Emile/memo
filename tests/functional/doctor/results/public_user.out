[ERROR] User:
  Reason: user "bob" has no private key
[OK] Storage resources
[OK] Networks
[OK] Volumes
[OK] Drives
[OK] Leftovers
